docs/source/sysadmin_guide/setting_allowed_txns.rst

```diff
@@ -0,0 +1,80 @@
+************************************************
+Setting the Allowed Transaction Types (Optional)
+************************************************
+
+By default, a validator accepts transactions from any transaction processor.
+However, Sawtooth allows you to limit the types of transactions that can be
+submitted.
+
+This setting, ``sawtooth.validator.transaction_families``, improves the
+Sawtooth network's security by ignoring any unrecognized transaction processors.
+It is an on-chain setting, which means that the change is submitted on one node;
+the other nodes in the network apply the settings change when they receive the
+block with this transaction. Note that the
+:doc:`Settings transaction processor <../transaction_family_specifications/settings_transaction_family>`
+is required to handle on-chain configuration settings.
+
+In this procedure, you will configure the validator network to limit the
+accepted transaction types to those from the Identity, IntegerKey, Settings, and
+PoET Validator Registry transaction processors.
+
+.. important::
+
+   For the environment described in this guide, you  **must** run this procedure
+   on the same node that created the genesis block, because the ``sawset
+   proposal create`` command requires the validator key that was generated on
+   that node.
+
+#. Open a terminal window on the "genesis node" (the Sawtooth node that created
+   the genesis block in a previous procedure).
+
+#. Use the ``sawset`` command to create and submit a batch of transactions that
+   changes the allowed transaction types.
+
+   .. code-block:: console
+
+      # sawset proposal create --key /etc/sawtooth/keys/validator.priv \
+      sawtooth.validator.transaction_families='[{"family":"sawtooth_identity", "version":"1.0"}, {"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}, {"family":"sawtooth_validator_registry", "version":"1.0"}]'
```

Comment:
There are spaces in the intkey chunk that are not in the others, but I am not sure if that matters.

Reply:
Good catches - thanks!